Content.Server/Atmos/EntitySystems/AtmosphereSystem.HighPressureDelta.cs

```diff
@@ -154,29 +152,39 @@ private void ConsiderPressureDifference(GridAtmosphereComponent gridAtmosphere,
         tile.PressureDirection = differenceDirection;
     }
 
-    public void ExperiencePressureDifference(
-        Entity<MovedByPressureComponent> ent,
+    public void ExperiencePressureDifference(Entity<MovedByPressureComponent> ent,
         int cycle,
-        float pressureDifference,
-        Vector2 throwVector,
+        Vector2 pressureVector,
+        ContentTileDefinition tile,
+        GravityComponent gravity,
+        EntityQuery<ProjectileComponent> projectileQuery,
+        float frameTime,
         TransformComponent? xform = null,
         PhysicsComponent? physics = null)
     {
         var (uid, component) = ent;
         if (!Resolve(uid, ref physics, false)
             || !Resolve(uid, ref xform)
             || physics.BodyType == BodyType.Static
-            || float.IsPositiveInfinity(component.MoveResist))
+            || float.IsPositiveInfinity(component.MoveResist)
+            || physics.LinearVelocity.Length() >= SpaceWindMaxVelocity)
```

Comment:
Doesn't this prevent higher velocity objects (even if they're moving *against* the suggested wind direction) from being affected by pressure difference?